AdtSampleApp/SampleClientApp/CommandLoop.cs

```diff
@@ -536,14 +534,14 @@ public async Task CommandCreateRelationship(string[] cmd)
             {
                 for (int i = 0; i < args.Length; i += 3)
                 {
-                    relationship.CustomProperties.Add(args[i], ConvertStringToType(args[i + 1], args[i + 2]));
+                    relationship.Properties.Add(args[i], ConvertStringToType(args[i + 1], args[i + 2]));
```

Comment:
This is hard to read but I understand if it's not in the scope of changes currently.